osu.Game.Rulesets.Osu/Objects/Spinner.cs

```diff
@@ -51,6 +51,11 @@ public double EndTime
         /// </summary>
         private const int bonus_spins_gap = 2;
 
+        /// <summary>
+        /// The maximum RPM at which HP will be restored.
+        /// </summary>
+        private const int hp_ticks_per_minute = 500;
```

Comment:
I can't *not* point out that this _still_ means there is a cap on spinning speed that you can reach past which you will not recover hp anymore. Whether this is acceptable I don't know.

Reply:
After changing the way HP ticks are triggered, it should be okay, as you can no longer drain all the hp ticks by spinning faster than this before completion.
Value here is sort of arbitrary, but feels pretty okay imo.

Reply:
> After changing the way HP ticks are triggered

What does this mean?

Reply:
Refers to a previous iteration of the logic in an older review.